src/app/common/subscriptions/SubscriptionDetails.vue

```diff
@@ -109,24 +109,24 @@ const statuses = computed<StatusRow[]>(() => {
 
 <style lang="scss" scoped>
 .overview-tertiary-title {
-  font-size: var(--type-sm);
-  font-weight: var(--font-weight-semi-bold);
-  color: var(--grey-500);
-  margin: var(--spacing-xs) 0;
+  font-size: $kui-font-size-30;
+  font-weight: $kui-font-weight-semibold;
+  color: $kui-color-text-neutral;
+  margin: $kui-space-40 0;
```

Comment:
```suggestion
  margin: $kui-space-40 $kui-space-0;
```

Reply:
What’s the purpose using a variable for 0 values?

Reply:
I think just so that styles can eventually all be tokenized. 

0=0 🤷

Reply:
For 0 values, that’s counterproductive. I can’t think of a situation in which one would change `$kui-space-0` to have any other value than `0`. The token name is explicitly not semantic (e.g. it’s not `$kui-space-lowest`). I’m leaving those instances as-is.

Reply:
This defeats the purpose; ideally every value going forward will be tokenized once we begin deeper integration.

Reply:
> This defeats the purpose

How? There is no point in tokenizing a value that’s both never changing and trivial to use as a literal value.

Reply:
Just giving you the agreed upon info.